### PR TITLE
chore: updated deps

### PR DESCRIPTION
**Pull request by zMotivat0r**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - '10'
+  - '14'
```
Comment (bashleigh) [inline .travis.yml:4]: Can I suggest switching to github actions? Travis seems really slow recenty and is currently switching from travis.org to their .com domain. Let me know if you'd like some help!

**Review by bashleigh — COMMENTED** [review]
Just a suggestion on travis vs github actions

File: .travis.yml (hunk 1 above)
Comment (zMotivat0r) [reply .travis.yml:4]: yep, I would love to have your help here, @bashleigh 
do you have some spare time today maybe?

Comment (zMotivat0r) [reply .travis.yml:4]: I'll merge this for now, btw

Comment (bashleigh) [reply .travis.yml:4]: sure, we can use this as a guide https://github.com/bashleigh/typeorm-polymorphic/blob/master/.github/workflows/main.yml

**Pull request merged**